packages/mui-system/src/createStyled/createStyled.js

```diff
@@ -1,4 +1,7 @@
-import styledEngineStyled, { internal_mutateStyles as mutateStyles } from '@mui/styled-engine';
+import styledEngineStyled, {
+  internal_mutateStyles as mutateStyles,
+  internal_css as css,
+} from '@mui/styled-engine';
```

Comment:
One thing to be aware of `@mui/styled-engine` is the consistency exports with `@mui/styled-engine-sc`. @mnajdova I believe that both of the packages must export the same function names right?

Reply:
Yes I'm aware of that, just touched this one for prototyping.